cmd/init.go

```diff
@@ -80,3 +87,83 @@ func initStatsClient() {
 
 	log.AddHook(hooks.NewLogrusHook(statsClient, globalConfig.Stats.ErrorsSection))
 }
+
+func initStatsExporter() {
+	var err error
+	logger := log.WithField("stats.exporter", globalConfig.Stats.Exporter)
+
+	// Register stats exporter according to config
+	switch globalConfig.Stats.Exporter {
+	case "datadog":
+	case "stackdriver":
+		logger.Warn("Not implemented!")
+		return
+	case "prometheus":
+		err = initPrometheusExporter()
+		break
+	default:
+		logger.Info("Unsupported or invalid stats exporter was specified")
+		return
+	}
+
+	if err != nil {
+		logger.Warn("Failed initialising stats exporter")
```

Comment:
probably you can add `.WithError(err)` here, just to have all the info in the single log entry, even if you already logged it previously